airflow/exceptions.py

```diff
@@ -113,33 +113,42 @@ class AirflowFailException(AirflowException):
     """Raise when the task should be failed without retrying."""
 
 
-class AirflowExecuteWithInactiveAssetExecption(AirflowFailException):
+class _AirflowExecuteWithInactiveAssetExecption(AirflowFailException):
```

Comment:
Looks like we don't want anyone to use it directly. We could probably add this to the docstring.

Reply:
I mean in general we don’t want the user to catch most (any?) exceptions here anyway?

Reply:
And the underscore prefix is a pretty clear sign to not use it directly

Reply:
Yep, I just worried contributors might use it somewhere (most likely not, but...)